src/test/java/org/reaktivity/reaktor/test/internal/k3po/ext/behavior/NukleusChannel.java

```diff
@@ -397,7 +397,8 @@ public int writableBytes(
         if (writableBytes > 0 && debitor != null && debitorIndex != -1L)
         {
             final int requiredBytes = writableBytes + writablePadding;
-            writableBytes = Math.max(debitor.claim(debitorIndex, targetId, requiredBytes, requiredBytes) - writablePadding, 0);
+            writableBytes = Math.max(debitor.claim(debitorIndex, targetId, requiredBytes,
+                requiredBytes) - writablePadding, 0);
```

Comment:
Is this a real diff or just formatting?  If formatting, please undo.

Reply:
it is real because of 130 characters.